provers/celestia-prover/prover/src/main.rs

```diff
@@ -73,34 +74,51 @@ impl Prover for ProverService {
         let client_id = inner_request.client_id.parse::<Address>().map_err(|e| {
             Status::internal(format!("Failed to parse client_id as EVM address: {}", e))
         })?;
+        println!("client_id: {:?}", client_id);
```

Comment:
Note to reviewers: we can remove a lot of the `println!` debug statements in this file. Up to reviewer's discretion.